Potapov Vladislav/task8.js

```diff
@@ -0,0 +1,7 @@
+function sum(...args) {
+    return args.length > 1
+        ? args[0] + sum(...args.slice(1))
```

Comment:
@TheUnforgivenOne Please pay attention to the main task - we shouldn't use es6, so, rest and spread operators are prohibited ;)